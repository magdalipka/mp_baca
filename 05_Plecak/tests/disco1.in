1
100000
100
13491 54395 46203 61208 92210 26569 73081 80991 91660 64639 60440 76256 4540 14456 38076 27532 23892 23341 47028 4073 96974 25602 5281 43412 53653 75050 11805 6892 41360 19728 39011 57746 74084 80327 20503 23533 93518 40704 91494 97798 68404 74185 10077 93057 59598 15498 2016 8030 77097 40569 18301 12693 67274 41145 37185 30358 37978 27660 21078 25010 28278 41061 5492 56055 62267 80093 98068 26446 29512 76470 52913 41826 47174 3109 12848 80644 3846 8045 66051 27836 26804 25614 41443 20067 51906 63136 14546 50572 28375 20621 88187 81730 43490 82476 39105 7500 21289 49374 95095 97048
