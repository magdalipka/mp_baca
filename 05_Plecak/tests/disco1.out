REC:  100000 = 13491 46203 4540 5281 10077 2016 3846 14546
ITER: 100000 = 13491 46203 4540 5281 10077 2016 3846 14546
